receiver/k8sobjectsreceiver/receiver.go

```diff
@@ -50,17 +51,21 @@ func newReceiver(params receiver.Settings, config *Config, consumer consumer.Log
 		return nil, err
 	}
 
-	for _, object := range config.Objects {
-		object.exclude = make(map[apiWatch.EventType]bool)
-		for _, item := range object.ExcludeWatchType {
-			object.exclude[item] = true
+	objects := make([]*K8sObjectsConfig, len(config.Objects))
+	for i, obj := range config.Objects {
+		copied := *obj // Copy the object
```

Comment:
`gvr` pointer field would need a deep copy but we can leave it as is for now and file an issue for further improving this code-base. I think we will need to completely move the mutable fields out of the Config struct into a new struct  type.

Reply:
@ChrsMark if we dont do this suggestion now can you file an issue for this desire.